src/main/kotlin/io/github/mojira/arisa/modules/KeepPrivateModule.kt

```diff
@@ -23,9 +23,9 @@ class KeepPrivateModule(
             val markedTime = comments.first(::isKeepPrivateTag).created
             val changedTime = changeLog.lastOrNull(::isSecurityChange)?.created
             if (changedTime != null && changedTime.isAfter(markedTime)) {
-                addComment(CommentOptions(message)).toFailedModuleEither().bind()
+                addComment(CommentOptions(message))
```

Comment:
add comment should be always after resolving, and it should only happen if the module didnt fail to do the resolve